rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/ValidationUtil.java

```diff
@@ -106,6 +110,19 @@ private static JobValidationData validateJob(String jobFilePath) {
 
     }
 
+    private static void fillUpdatedVariables(TaskFlowJob job, JobValidationData data) {
+        HashMap<String, String> updatedVariables = new HashMap<>();
+        for (JobVariable jobVariable : job.getVariables().values()) {
+            updatedVariables.put(jobVariable.getName(), jobVariable.getValue());
+        }
+        for (Task task : job.getTasks()) {
+            for (TaskVariable taskVariable : task.getVariables().values()) {
+                updatedVariables.put(task.getName() + ":" + taskVariable.getName(), taskVariable.getValue());
+            }
+        }
+        data.setUpdatedVariables(updatedVariables);
```

Comment:
is it possible that data.updatedVariables already contains some value before this point? Basically is there the risk to loose/reset information here?